corehq/apps/sms/models.py

```diff
@@ -2297,3 +2290,6 @@ def has_migration_completed(cls, name):
             return True
         except cls.DoesNotExist:
             return False
+
+
+from corehq.apps.sms.signals import *
```

Comment:
actually now that i think about it all we really need to do is import the module to register the signals in that file, so i updated it to do that (this way we don't import *)